src/store/modules/current-company/__tests__/actions.test.js

```diff
@@ -0,0 +1,13 @@
+import { updateCompany } from "store/modules/current-company/actions";
+
+describe('Current company actions', () => {
+  const mockPayload = {id: 1, name: 'Test'}
+  it('#updateCompany', () => {
+    expect(updateCompany(mockPayload)).toStrictEqual(
+      {
+        payload: {id: 1, name: "Test"},
+        type: 'currentCompany/SET_CURRENT_COMPANY',
+      }
+    )
+  })
+})
```

Comment:
In order to improve the readability of the code, we could separate some steps of the test with variables, instead of doing a lot of things in the expect directly. Something like:
```suggestion
import { updateCompany } from 'store/modules/current-company/actions';

describe('Current company actions', () => {
  it('#updateCompany', () => {
    const mockPayload = { id: 1, name: 'Test' };
    const expectResult = {
      payload: { id: 1, name: 'Test' },
      type: 'currentCompany/SET_CURRENT_COMPANY',
    };

    const result = updateCompany(mockPayload);

    expect(result).toStrictEqual(expectResult);
  });
});
```

Does that make sense?